sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -643,8 +685,20 @@ def from_runner_api(cls, logical_type_proto):
     if logical_type is None:
       raise ValueError(
           "No logical type registered for URN '%s'" % logical_type_proto.urn)
-    # TODO(bhulette): Use argument
-    return logical_type()
+    if not logical_type_proto.HasField(
+        "argument_type") or not logical_type_proto.HasField("argument"):
+      # logical type_proto without argument
+      return logical_type()
+    else:
+      try:
+        argument = value_from_runner_api(
+            logical_type_proto.argument_type, logical_type_proto.argument)
+      except ValueError:
+        # TODO(https://github.com/apache/beam/issues/23373): Complete support
+        # for logical types that require arguments beyond atomic type.
+        # For now, skip arguments.
+        return logical_type()
```

Comment:
I think it would be better to fail loudly here. It's dangerous to move forward with a type we don't understand.

Are there cases where we need to let this through?

Reply:
Yeah, this was the behavior before (arguments omitted) and the fixed_decimal logical type has a Row type argument. It is special because the `scale` argument is encoded both in argument and in its representation, so that I do not need the argument to recover the data from bytes. The argument instead add a guard for overflow (like for fixed_char and fixed_bytes) here. Since the same check is done in Java side so it is safe so far.

Added a warning here.